File: TP2-Java/Program Java/Elektronik.java
// File: Elektronik.java
// Kelas Produk diasumsikan sudah ada dan berada dalam direktori yang sama.
// Jika tidak, perlu diimport.

public class Elektronik extends Produk {
    // Atribut tambahan untuk kelas Elektronik
    protected String lokasi_penyimpanan;
    protected String nama;
    protected String merk;

    // Konstruktor
    public Elektronik(String id_produk, String asal_produk, double harga, String lokasi_penyimpanan, String nama, String merk) {
        // Memanggil konstruktor dari kelas induk (Produk)
        super(id_produk, asal_produk, harga);
        this.lokasi_penyimpanan = lokasi_penyimpanan;
        this.nama = nama;
        this.merk = merk;
    }

    // Getter methods
    public String getLokasi_penyimpanan() {
        return lokasi_penyimpanan;
    }

    public String getNama() {
        return nama;
    }

    public String getMerk() {
        return merk;
    }

}

File: TP2-Python/TP2-Kode Program/elektronik.py
from produk import Produk

class Elektronik(Produk):
    def __init__(self, id_produk, asal_produk, harga, lokasi_penyimpanan, nama, merk):
        super().__init__(id_produk, asal_produk, harga)
        self._lokasi_penyimpanan = lokasi_penyimpanan
        self._nama = nama
        self._merk = merk

    # Getter methods
    def get_lokasi_penyimpanan(self):
        return self._lokasi_penyimpanan

    def get_nama(self):
        return self._nama

    def get_merk(self):
        return self._merk

  